docs/Passing-Devices-with-Clear-Containers.md

```diff
@@ -29,6 +29,15 @@ to `vfio-pci` driver and then pass the IOMMU group that the device belongs to on
 the docker command line.
 For detailed steps to bind and unbind devices, see https://github.com/containers/virtcontainers#how-to-pass-a-device-using-vfio-passthrough
 
+You can verify that the device is passed to the Clear Containers container by running
+`lspci` command inside the container.
+
+For the device to be visible inside the container under /dev, you need to pass /dev
+while starting a container as shown below:
+
+```
+$ sudo docker run -it -v /dev:/dev --device=/dev/vfio/16 centos/tools bash
```

Comment:
I think this should be combined with the initial CLI example, and then perhaps explained as you did above.  

There's no usefulness in passing it into the VM, but not the container, after all.

Reply:
@amshinde per OCI spec, the devices are container specific resources. So this approach will work fine for docker containers. How do we plan to handle this for kubernetes PODs, where the devices may be spread across containers (i.e. each device belongs to a different container for example)

Reply:
@mcastelino Containers in a k8 pod should share the same mount namespace eventually.